yt_dlp/extractor/rheinmaintv.py

```diff
@@ -0,0 +1,601 @@
+from ..utils import extract_attributes, float_or_none, int_or_none, parse_iso8601
+from .common import InfoExtractor
+
+
+class RheinMainTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rheinmaintv\.de/sendungen/(?:[a-z-]+/)*(?P<display_id>[a-z-]+)/vom-(?P<date>[0-9]{2}\.[0-9]{2}\.[0-9]{4})(?:/(?P<serial_number>[0-9]+))?'
+    _TESTS = [{
+        'url': 'https://www.rheinmaintv.de/sendungen/beitrag-video/formationsgemeinschaft-rhein-main-bei-den-deutschen-meisterschaften/vom-14.11.2022/',
+        # 'md5': ...,  # left out because checksum is changing
```

Comment:
```suggestion
```
This is expected for fragmented formats

Reply:
Fine.
You might consider improving the extractor template. Otherwise, everybody will have to figure that out over and over again.